data-table-cell.html

```diff
@@ -58,6 +80,15 @@
                   '_orderChanged(order)',
                   '_widthChanged(width)'],
 
+      listeners: {
+        'dragstart': '_handleDragStart',
```

Comment:
I think it would be better to add these listeners only when `reorderable` is set to `true`. Saves making all those checks in the the handlers. Also, I've used the naming pattern `_onEventName` for event handlers.